net-mgmt/configsync/src/opnsense/mvc/app/views/VerbNetworks/ConfigSync/settings.volt

```diff
@@ -0,0 +1,206 @@
+        font-family: Menlo, Monaco, Consolas, "Courier New", monospace;
+    }
+</style>
+
+<script>
+    
+    /**
+     * updateFullURI
+     */
+    function updateFullURI() {
+        if($('#configsync\\.settings\\.Provider').val() === 'awss3') {
+            var link_url = 'https://s3.console.aws.amazon.com/s3/buckets/';
+            link_url += $('#configsync\\.settings\\.StorageBucket').val() + '/';
+            link_url += $('#configsync\\.settings\\.StoragePath').val() + '/';
+
+            var link_name = 's3://';
+            link_name += $('#configsync\\.settings\\.StorageBucket').val() + '/';
+            link_name += $('#configsync\\.settings\\.StoragePath').val();
+        }
+        $('#configsync\\.settings\\.StorageProviderLink').html('<a target="_blank" href="' + link_url +'">' + link_name + '</a>');
+    }
+    
+    /**
+     * $(document).ready
+     */
+    $(document).ready(function() {
+        
+        updateServiceControlUI('configsync');
+        
+        $("#configsync\\.settings\\.StorageBucket").change(function(){
+            updateFullURI();
+        });
+        
+        $("#configsync\\.settings\\.StoragePath").change(function(){
+            updateFullURI();
+        });
+        
+        mapDataToFormUI({frm_Settings:"/api/configsync/settings/get"}).done(function(data){
+            formatTokenizersUI();
+            $('.selectpicker').selectpicker('refresh');
+            updateFullURI();
+        });
+
+        $("#testAction").click(function(){
+            $("#responseMsg").removeClass("hidden").removeClass("alert-danger").addClass('alert-info').html("Running tests...");
+            saveFormToEndpoint(
+                url='/api/configsync/settings/test', 
+                formid='frm_Settings', 
+                callback_ok=function(data){
+                    if(data['status'] != 'success') {
+                        $("#responseMsg").removeClass("alert-info").addClass('alert-danger');
+                    }
+                    $("#responseMsg").html(data['message']);
+                },
+                disable_dialog=true
+            );
+        });
+        
+        $("#saveAction").click(function(){
+            $("#responseMsg").removeClass("hidden").removeClass("alert-danger").addClass('alert-info').html("Saving settings...");
+            saveFormToEndpoint(
+                url='/api/configsync/settings/set', 
+                formid='frm_Settings', 
+                callback_ok=function(){
+                    $("#responseMsg").html("Configuration Sync service settings saved.");
+                    ajaxCall(url="/api/configsync/service/reload", sendData={}, callback=function(data, status) {
+                        $("#responseMsg").html("Configuration Sync service settings saved and reloaded.");
+                    });
+                },
+                disable_dialog=true 
+            );
+            updateServiceControlUI('configsync');
+        });
+
+    });
+</script>
```

Comment:
the script should have spaces around the operators etc. like `a = b`

Reply:
Done